napari/components/add_layers_mixin.py

```diff
@@ -49,6 +49,7 @@ def add_layer(self, layer):
 
         if len(self.layers) == 1:
             self.reset_view()
+        return layer
```

Comment:
Should a test be added for this?

Reply:
I'm not sure,  it wasn't exactly an error that it didn't previously return a layer.  (since you already had the layer if you were calling this method in the first place)... And we didn't really need it to return one until I wanted it below in this PR :)

Reply:
I think no need